internal/pkg/template/env.go

```diff
@@ -124,8 +124,9 @@ type EnvOpts struct {
 type CDNConfig struct{}
 
 type VPCConfig struct {
-	Imported *ImportVPC // If not-nil, use the imported VPC resources instead of the Managed VPC.
-	Managed  ManagedVPC
+	Imported            *ImportVPC // If not-nil, use the imported VPC resources instead of the Managed VPC.
+	Managed             ManagedVPC
+	SecurityGroupConfig SecurityGroupConfig
```

Comment:
Same concern! Whether we consider environment security group a part of VPC config or not 🤔